tracer/src/Datadog.Trace/Iast/StackWalker.cs

```diff
@@ -31,18 +31,20 @@ internal static class StackWalker
         "Oracle.ManagedDataAccess",
         "System.Data",
         "System",
+        "System.Configuration.ConfigurationManager",
         "System.Core",
         "System.Linq",
+        "System.Net.Security",
```

Comment:
you re-ordered alphabetically except for this one?

Reply:
Ups, I missed that one...